doc/tutorial/get_started.md

```diff
@@ -66,10 +66,22 @@ MicroCloud requires LXD version 5.21:
 
      Enter `40`.
 
+   - `What IPv4 address should be used? (CIDR subnet notation, “auto” or “none”)`
+
+     Enter `10.1.123.1/24`.
+
+   - `What IPv6 address should be used? (CIDR subnet notation, “auto” or “none”)`
+
+     Enter `fd42:1:1234:1234::1/64`.
+
    - `Would you like the LXD server to be available over the network? (yes/no)`
 
      Enter `yes`.
 
+1. Modify the default network so we can later define specific IPv6 addresses for the VMs:
+
+       lxc network set lxdbr0 ipv6.dhcp.stateful true
+
```

Comment:
````suggestion

```{note}
In the steps above, we ask you to specify the IP addresses to be used instead of accepting the defaults. While this is not strictly required for this setup, it causes the example IPs displayed in this tutorial to match what you see on your system, which improves clarity.
```

````